orchestrator/proposer/proposer.proto

```diff
@@ -5,136 +5,144 @@ package proposer;
-  optional string forwarder_address = 7;
+    message P2PKeyBundle {
+        string peer_id    = 1;
+        string public_key = 2;
+    }
+
+    message OCRKeyBundle {
+        string bundle_id               = 1;
+        string config_public_key       = 2;
+        string offchain_public_key     = 3;
+        string onchain_signing_address = 4;
+    }
+
+    message Plugins {
+        bool commit     = 1;
+        bool execute    = 2;
+        bool median     = 3;
+        bool mercury    = 4;
+        bool rebalancer = 5;
+    }
+
+    bool enabled                      = 1;
+    bool is_bootstrap                 = 2;
+    P2PKeyBundle p2p_key_bundle       = 3;
+    OCRKeyBundle ocr_key_bundle       = 4;
+    string multiaddr                  = 5;
+    Plugins plugins                   = 6;
+    optional string forwarder_address = 7;
 }
 
 message ChainConfig {
-  Chain chain = 1;
-  string account_address = 2;
-  string admin_address = 3;
-  FluxMonitorConfig flux_monitor_config = 4;
-  OCR1Config ocr1_config = 5;
-  OCR2Config ocr2_config = 6;
+    Chain chain                           = 1;
+    string account_address                = 2;
+    string admin_address                  = 3;
+    FluxMonitorConfig flux_monitor_config = 4;
+    OCR1Config ocr1_config                = 5;
+    OCR2Config ocr2_config                = 6;
+    // For EVM chains, we do not need this value and it is kept in the node's
+    // keystore. For starknet, because the wallet address needs to be deployed
+    // using this value and this pub key needs to be passed into the starknet
+    // relayer, we request the node to send this directly to CLO.
+    optional string account_address_public_key = 7;
 }
 
 message UpdateNodeRequest {
-  repeated JobType job_types = 1;
-  int64 chain_id = 2; // To be removed when all nodes are upgraded to 1.2
-  repeated string account_addresses = 3;
-  bool is_bootstrap_peer = 4;
-  string bootstrap_multiaddr = 5;
-  string version = 6;
-  repeated int64 chain_ids = 7;
-  repeated Account accounts = 8;
-  repeated Chain chains = 9;
-
-  repeated ChainConfig chain_configs = 10;
+    repeated JobType job_types = 1;
+    int64 chain_id = 2;  // To be removed when all nodes are upgraded to 1.2
+    repeated string account_addresses = 3;
+    bool is_bootstrap_peer            = 4;
+    string bootstrap_multiaddr        = 5;
+    string version                    = 6;
+    repeated int64 chain_ids          = 7;
+    repeated Account accounts         = 8;
+    repeated Chain chains             = 9;
+
+    repeated ChainConfig chain_configs = 10;
 }
 message UpdateNodeResponse {}
 
 message ApprovedJobRequest {
-  string uuid = 1;
-  int64 version = 2;
+    string uuid   = 1;
+    int64 version = 2;
 }
 message ApprovedJobResponse {}
 
 message HealthcheckRequest {}
 message HealthcheckResponse {}
 
 message RejectedJobRequest {
-  string uuid = 1;
-  int64 version = 2;
+    string uuid   = 1;
+    int64 version = 2;
```

Comment:
hmm why the extra 2 space on all the fields ?

Reply:
https://github.com/smartcontractkit/chainlink-protos/blob/518dfaccf9acc5a614a01181bd4252a87d38d6f4/.editorconfig#L14 just a bit of formatting. Seems that the tab spacing was set to 4 spaces